src/main.js

```diff
@@ -77,59 +79,49 @@ function submitData(){
 
 function displayTimer() {
   newActivity.innerText = 'Current Activity'
-  var display = '04:00'
+  var min = parseInt(currentActivity.minutes);
+  var sec = parseInt(currentActivity.seconds);
+  min = min <10 ? '0' + min : min
+  sec = sec <10 ? '0' + sec : sec
   activityContainer.innerHTML = `
-  <div id='testID'>Button </div>
-    <a href="" class="btn btn-start">START</a>
-    <button id='startBtn'>Start</button>
+    <div>${min}:${sec}</div>
+    <a href="" id="btn" class="btn btn-start">START</a>
+    
     `
 }
 
-// var startBtn = document.querySelector('#')
-
-activityContainer.addEventListener('click', updateTimer)
-
-function updateTimer() {
-  if(event.target.id === 'startBtn') {
-    var startMin = 5;
-    var startSec = 20;
+function updateTimer(event) {
+  if(event.target.id === 'btn') {
+    var startMin = parseInt(currentActivity.minutes);
+    var startSec = parseInt(currentActivity.seconds);
     var time = startMin*60 + startSec
     setInterval(function() {
       var min = Math.floor(time/60)
       var sec = time % 60
-
+      
       min = min <10 ? '0' + min : min
       sec = sec <10 ? '0' + sec : sec
-
-      activityContainer.innerHTML = `${min}: ${sec}`
-
+      
+      activityContainer.innerHTML = `
+      ${min}:${sec}
+      <a href="" id="btn" class="btn btn-start">START</a>
```

Comment:
This looks great! I was wondering how we were going to get it into the HTML. Good job!